admin/admin.js

```diff
@@ -122,6 +122,41 @@ jQuery(function($){
         $(window).resize();
     });
 
-    // Finally enable css3 animations on the widgets list
+    // Enable css3 animations on the widgets list
     $('#widgets-list').addClass('so-animated');
-});
\ No newline at end of file
+
+    // Handle the dialog
+    var dialog = $('#sow-settings-dialog');
```

Comment:
Not really. Most of this code is fairly specific to the Widgets Bundle settings dialog. It submits to an iframe while the Page Builder dialog stores to a JS object. Similar, but it would be very difficult to reuse.